.github/file-paths.yaml

```diff
@@ -84,8 +84,8 @@ snowplow:
 
 documentation:
   - "docs/**"
-  - "**.md"
+  - "**/*.md"
```

Comment:
Click to see results using the 3rd party tool:

1. [Before](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A%2A.md&matches=false&options=dot%3Atrue&tests=foo.md&tests=foo%2Fbar.md)
2. [After](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A%2A%2F%2A.md&matches=false&options=dot%3Atrue&tests=foo.md&tests=foo%2Fbar.md)